AdrevenueTestApp/AdrevenueTestApp/ViewController.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  ViewController.swift
+//  TestAppForAdrevenue
+//
+//  Created by Moris Gateno on 06/08/2023.
+//
+
+import UIKit
+import AppsFlyerAdRevenue
+import AppsFlyerLib
+
+class ViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+    }
+
+    @IBAction func button(_ sender: Any) {
+        let dictionary = NSMutableDictionary()
```

Comment:
Why don't you use regular Dictionary? There is no need to init NSDictionary and cast it.